mutyper/cli.py

```diff
@@ -165,8 +167,14 @@ def spectra(args):
         spectra_data = defaultdict(lambda: np.zeros_like(vcf.samples,
                                                          dtype=int))
 
-        for variant in vcf:
-            spectra_data[variant.INFO['mutation_type']] += variant.gt_types
+        if args.randomize:
+            for variant in vcf:
+            	curr_hap = choice([x for x, y in enumerate(variant.gt_types) 
+            						 for _ in range(y)])
+            	spectra_data[variant.INFO['mutation_type']][curr_hap] += 1.
```

Comment:
suggesting more descriptive variable name
```suggestion
            	random_haplotype = choice([x for x, y in enumerate(variant.gt_types) 
            						 for _ in range(y)])
            	spectra_data[variant.INFO['mutation_type']][random_haplotype] += 1.
```